services/net/transcription/appsettings.json

```diff
@@ -66,5 +67,11 @@
       "BatchSize": 16384,
       "LingerMs": 1
     }
+  },
+  "S3": {
+    "BucketName": "${S3__BucketName}",
```

Comment:
Why set these to invalid values?

Reply:
I think these values are from .env file locally, they are valid, I think.

Reply:
This will set the S3.BucketName to equal `"${S3__BucketName}"`, which is not valid.  It isn't clear what you are doing here.  Are you adding a form of documentation?